HousingRepairsOnlineApi/SoRConfig.json

```
@@ -156,6 +156,13 @@
   },
   "outside": {
     "securityLights": "N880112",
+    "door": {
+      "shedDoor": "N328101",
+      "outhouseCupboardDoor": "N330001",
+      "woodenBackDoor": "N321305",
+      "UPVCBackDoor": "N325117",
```

To stick with the naming convention:
```suggestion
      "upvcBackDoor": "N325117",
```